approvals/src/test/java/no/sikt/nva/approvals/domain/ApprovalServiceTest.java

```diff
@@ -224,4 +229,62 @@ void shouldThrowApprovalServiceExceptionWhenRepositoryFailsToCheckExistingIdenti
 
         assertThrows(ApprovalServiceException.class, () -> approvalService.create(identifiers, randomUri()));
     }
+
+    @Test
+    void shouldUpdateApprovalIdentifiersSuccessfully()
+        throws RepositoryException, ApprovalNotFoundException, ApprovalServiceException, ApprovalConflictException {
```

Comment:
If you extract the things that can throw exceptions to helper methods with a try/catch, you wouldn't need to specify `throws` in the signature of every test. Would be a bit more readable I think.